CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 2.2.2
```

Comment:
To respect semantic versioning, it's not gonna be 2.2.2 but 2.3.0 as it is not a bug fix release but adds new APIs.

(Also pls add newline after markdown section title)